core/src/main/scala/com/databricks/labs/remorph/graph/TableGraph.scala

```diff
@@ -13,6 +13,9 @@ class TableGraph(parser: PlanParser[_]) extends DependencyGraph with LazyLogging
   private val nodes = scala.collection.mutable.Set.empty[Node]
   private val edges = scala.collection.mutable.Set.empty[Edge]
 
+  def getNodes: Set[Node] = nodes.toSet
+  def getEdges: Set[Edge] = edges.toSet
```

Comment:
```suggestion
  private[graph] val nodes = scala.collection.mutable.Set.empty[Node]
  private[graph] val edges = scala.collection.mutable.Set.empty[Edge]
```

create `GraphvizGenerator` class in the same package as `TableGraph` and use these package-private properties.